test/conformance/ptp/ptp.go

```diff
@@ -356,14 +370,102 @@ var _ = Describe("[ptp]", func() {
 							buf, _ := pods.ExecCommand(client.Client, pod, PtpContainerName, []string{"curl", "127.0.0.1:9091/metrics"})
 							return buf.String()
 						}, 5*time.Minute, 5*time.Second).Should(ContainSubstring("openshift_ptp_offset_ns"),
-							fmt.Sprint("Time metrics are not detected"))
+							"Time metrics are not detected")
 						slavePodDetected = true
 						break
 					}
 				}
 				Expect(slavePodDetected).ToNot(BeFalse(), "No slave pods detected")
 			})
 		})
+
+		Context("Running with event enabled", func() {
+			ptpSlaveRunningPods := []v1core.Pod{}
+			BeforeEach(func() {
+				if !ptpEventEnabled() {
+					Skip("Skipping, PTP events not enabled")
+				}
+				if isSingleNode {
+					Skip("Running in single node mode")
+				}
+				if discoveryFailed {
+					Skip("Failed to find a valid ptp slave configuration")
+				}
+				ptpPods, err := client.Client.Pods(PtpLinuxDaemonNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: "app=linuxptp-daemon"})
+				Expect(err).NotTo(HaveOccurred())
+				Expect(len(ptpPods.Items)).To(BeNumerically(">", 0), "linuxptp-daemon is not deployed on cluster")
+
+				for _, pod := range ptpPods.Items {
+					if podRole(pod, slaveNodeLabel) {
+						waitUntilLogIsDetected(pod, 3*time.Minute, "Profile Name:")
+						ptpSlaveRunningPods = append(ptpSlaveRunningPods, pod)
```

Comment:
any reason for this to not run on ptp masters?

Reply:
I ignored it because I am interested in slave configured events only and I am merely validating event functionality. 
technically one node is good test . 
Also I am not validating  the data itself , just the functionality and slave logs are more valid for my test case . 
what do you think ? Should I enhance it to include master too?

Reply:
Also it will be faster just checking one pod .